yast/yast-storage-ng
# add attributes to bcache

Manual testing showed problems with reboot:

1. cache_mode works as expected after reboot
2. sequential cutoff and writeback_percent is reset.

Another problem is really rename in memory which result in failure as name is different then expected. Way to reproduce it is to create bcache0, then bcache1 and then delete bcache0.

## Conclusion

Rename was solved in libstorage-ng. For sequential cutoff and writeback percent as we do not get answer yet from Coly, we add only cache_mode, but commit to remove them can be easily reverted.

![snimek obrazovky_2018-10-18_17-55-32](https://user-images.githubusercontent.com/478871/47169930-11b26000-d305-11e8-95b4-dd76358b2873.png)
